data/src/config/server.rs

```diff
@@ -96,6 +100,8 @@ pub enum Sasl {
         username: String,
         /// Account password,
         password: String,
```

Comment:
Shouldn't this be changed to `Optional<String>` so we only need to fill one of them when using Plain Sasl?

Reply:
Made this Optional and check at runtime that one of password or password_file is set.